README.md

```diff
@@ -14,29 +8,6 @@ This repository includes files with plain SQL that can be used to recreate a dat
 
 <a name="readme-top"></a>
 
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
-
-Modify this file to match your project and remove sections that don't apply.
-
-REQUIRED SECTIONS:
-- Table of Contents
-- About the Project
-  - Built With
-  - Live Demo
-- Getting Started
-- Authors
-- Future Features
-- Contributing
-- Show your support
-- Acknowledgements
-- License
-
-After you're finished please remove all the comments and instructions!
--->
-
-
 <!-- TABLE OF CONTENTS -->
 
 # 📗 Table of Contents
```

Comment:
- [ ] Kindly remove the following links from the table of contents as they are broken because the corresponding sections have been removed.
![image](https://github.com/tsheporamantso/vet_clinic/assets/81932478/16054594-dbf8-4a43-bbb3-450e6e7cac67)
![image](https://github.com/tsheporamantso/vet_clinic/assets/81932478/be4d8b6c-3f3a-4c95-8634-add2af5bd33e)

Reply:
Removed

Reply:
well resolved! ✔️